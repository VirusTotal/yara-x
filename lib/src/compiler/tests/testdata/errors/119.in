rule test {
  strings: 
    $a = "aa" base64
  condition:
    $a
}