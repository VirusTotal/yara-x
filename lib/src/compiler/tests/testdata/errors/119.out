error: invalid pattern `$a`
 --> line:3:10
  |
3 |     $a = "aa" base64
  |          ^^^^ this pattern is too short
  |
  = note: `base64` requires that pattern is at least 3 bytes long